client/my-sites/stats/stats-email-top-row/top-card.jsx

```diff
@@ -29,13 +30,17 @@ const TopCardValue = ( { value, isLoading } ) => {
 	);
 };
 
-const TopCard = ( { heading, icon, value, isLoading } ) => {
+const TopCard = ( { heading, icon, value, isLoading, emailIsSending = false } ) => {
+	const translate = useTranslate();
 	return (
 		<Card className="highlight-card">
 			<div className="highlight-card-icon">{ icon }</div>
 			<div className="highlight-card-heading">{ heading }</div>
-			<div className="highlight-card-count">
-				<TopCardValue value={ value } isLoading={ isLoading } />
+			<div className={ `highlight-card-count ${ emailIsSending ? 'sending-email' : '' }` }>
```

Comment:
Nit but we use quite a lot of `is-` in modifier classnames like these around Calypso.

Reply:
Just updated to use is- prefix!